streams/src/main/java/org/apache/kafka/streams/internals/ApiUtils.java

```diff
@@ -30,7 +30,8 @@ private ApiUtils() {
      * @param name Name of params for an error message.
      * @return Milliseconds from {@code duration}.
      */
-    public static long validateMillisecondDuration(final Duration duration, final String name) {
+    public static long validateMillisecondDuration(final Duration duration, final String name)
+            throws IllegalArgumentException {
```

Comment:
nit: move to line above (however, this is an internal API, and I am not even sure if we need to add this at all.

Reply:
removed the `throws` since we are not catching exception caller side.